app/uploaders/art_photo_uploader.rb

```diff
@@ -0,0 +1,14 @@
+# encoding: utf-8
+
```

Comment:
Just remove that file and the test image file and the gem in the gem file and then this will be all set.